README.md

```diff
@@ -0,0 +1,84 @@
+# Clearable Token
+
+[![Build Status](https://travis-ci.org/IoBuilders/clearable-token.svg?branch=master)](https://travis-ci.org/IoBuilders/clearable-token)
+[![Coverage Status](https://coveralls.io/repos/github/IoBuilders/clearable-token/badge.svg?branch=master)](https://coveralls.io/github/IoBuilders/clearable-token?branch=master)
+[![npm](https://img.shields.io/npm/v/eip1996.svg)](https://www.npmjs.com/package/eip1996)
```

Comment:
In the links it needs to be changed from `eip1996` to `eip2018`

Reply:
sorry for that